src/_C009.cpp

```diff
@@ -135,7 +135,7 @@ bool do_process_c009_delay_queue(int controller_number, const Queue_element_base
         jsonString += F("\"ESP\":{");
         {
           // Create nested objects in "ESP":
-          jsonString += to_json_object_value(F("name"), Settings.Name);
+          jsonString += to_json_object_value(F("name"), Settings.getUnitname());
```

Comment:
Not sure what side-effects this may have, since the unit nr is also present.

Reply:
Existing method `getHostname()` includes the unit-number, `getUnitname()` only has the unit-name with any template values replaced.

Reply:
Renamed `getHostname()` to `getName()`